RenderStatePackager/src/Main.cpp

```diff
@@ -0,0 +1,131 @@
+/*
+ *  Copyright 2019-2021 Diligent Graphics LLC
```

Comment:
It is better to name the file "main" (without capital M) to be consistent with standard c/c++ notation